src/components/modals/warning-modal.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon, Modal } from 'semantic-ui-react';
+
+/**
+ * This wrapper draws the modal if
+ * redux state says that a Modal should be drawn
+ * otherwise it draws nothing.
+ */
+export const WarningModalWrapper = (props) => {
+  if (props.modal !== null && props.modal !== undefined) {
```

Comment:
This could probalby just be if (!props.modal)

Reply:
Yes.